packages/components/organisms/f-checkout/src/services/addressService.js

```diff
@@ -70,6 +72,25 @@ function getAddressClosestToPostcode (postcode, addresses) {
     return address;
 }
 
+const isAddressInLocalStorage = () => window.localStorage.getItem(storedLocationKey) !== null;
```

Comment:
You should check that window and window.localstorage exist first

window won't exist if the code happens to be run on the server - which it won't atm but good practice
I think incognito windows in some browsers disable localstorage, so worth checking it exists

Reply:
Done 🙂